<p>As defined by the specification, there can be at most one body parameter in each operation. This violation applies
to OpenAPI 2 only, a protection against it was added in the OpenAPI 3 format.

See https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#path-item-object.</p>

<h2>Noncompliant Code Example (OpenAPI 2)</h2>
<pre>
swagger: "2.0"
info:
  version: 1.0.0
  title: Swagger Petstore
paths:
  /pets:
    post:             # Noncompliant: An operation can have at most one body parameter.
      parameters:
      - in: body
        name: someParam
        required: true
        schema:
          type: string
      - in: body
        name: otherParam
        required: true
        schema:
          type: string
      responses:
        default:
          description: the default response
</pre>
<h2>Compliant Solution (OpenAPI 2)</h2>
<pre>
swagger: "2.0"
info:
  version: 1.0.0
  title: Swagger Petstore
paths:
  /pets:
    post:
      parameters:
      - in: body
        name: someParam
        required: true
        schema:
          type: string
      - in: query
        name: otherParam
        required: true
        schema:
          type: string
      responses:
        default:
          description: the default response
</pre>
