<p>The OpenAPI specification defines several places where mime types (for specification v2) or media type ranges (for
specification v3) can be used. The defined types must conform to the <a href="https://tools.ietf.org/html/rfc6838">RFC6838</a>
and <a href="https://tools.ietf.org/html/rfc7231">RFC7231</a>. This ensures that a tool
can rely on the declared media type to generate appropriate code to handle the parsing of output.</p>

<h2>Noncompliant Code Example</h2>

<h2>Compliant Solution</h2>
