<p>Define a summary for each operation. The OpenAPI contract is a way to convey meaning to non-technical users. The
summary will help those populations understand what the operation is about, instead of approaching it in technical terms.
</p>

<h2>Noncompliant Code Example (OpenAPI 2)</h2>
<pre>
swagger: "2.0"
info:
  version: 1.0.0
  title: Swagger Petstore
paths:
  /pets:
    post:             # Noncompliant: Provide a summary for each operation.
      responses:
        default:
          description: the default response
    get:
      summary: list all pets
      responses:
        default:
          description: the default response
</pre>
<h2>Compliant Solution (OpenAPI 2)</h2>
<pre>
swagger: "2.0"
info:
  version: 1.0.0
  title: Swagger Petstore
paths:
  /pets:
    post:
      summary: create a new pet
      responses:
        default:
          description: the default response
    get:
      summary: list all pets
      responses:
        default:
          description: the default response
</pre>
